### ISSUE TITLE
Error parsing JSON

### ISSUE THREAD

**Issue by Flint911jr** [ZcashFoundation/zec-qt-wallet#385]
Good afternoon. Please help me to sort out the issue. I'm trying to use the z_sendmany function, but it doesn't seem to work. I read the documentation before using it. Why does it result in an error? I work on Windows 10.

![Error parsing JSON](https://user-images.githubusercontent.com/108809453/188099071-7e732a48-d86e-4be3-bdf9-9b26301a9c5a.jpg)
![image](https://user-images.githubusercontent.com/108809453/188099726-f0db6d9a-bb8e-4a05-9f94-9b159159ec5c.png)

**Comment by alex138038** [comment]
if you want to send coins from "t address" to "t address", it is better to use the command: "sendtoaddress"

**Comment by Flint911jr** [comment]
Yes, sending to "t" address works fine. But I need to work with "z" address. It is not clear why the "z_sendmany" syntax does not work, because everything is exactly described in the API technical documentation.
How can you now transfer coins from "z" address to "t" address?

**Comment by Flint911jr** [comment]
Maybe you need to install zecwallet on linux and run commands in this environment, because zecwallet officially works this way?

**Comment by alex138038** [comment]
> Maybe you need to install zecwallet on linux and run commands in this environment, because zecwallet officially works this way?

I can't say for sure, because I don't work with "z addresses". 

Try using the linux command: z_sendmany "t1M72Sfpbz1BPpXFHz9m3CdqATR44Jvaydd" "[{\"address\": \"t1M72Sfpbz1BPpXFHz9m3CdqATR44Jvaydd\", \"amount\": 0.02 }]" 0 0.00002

**Comment by Flint911jr** [comment]
Some procedures do not work in the windows version of zecwallet. For example, procedures for working with z-addresses such as z_sendmany. Non-working procedures misinterpret arguments on the command line and throw an error like "Error parsing JSON".
If I understood everything correctly, this problem cannot be solved when working with windows, since changes have been made to the executable code of the application "zcash-cli.exe". In other words, some features of Zecwallet Fullnode are simply disabled.
There are also problems with the mobile version of the wallet.
The fully functional version of the wallet only works under Linux or similar operating systems. I was able to use z_sendmany under Linux.
All my attempts to contact the wallet developers, in particular with adityapk00, did not bring results.
As I understand it, support for the windows version of the wallet just suddenly ended without explanation.

**Comment by Flint911jr** [comment]
> > Maybe you need to install zecwallet on linux and run commands in this environment, because zecwallet officially works this way?
> 
> I can't say for sure, because I don't work with "z addresses".
> 
> Try using the linux command: z_sendmany "t1M72Sfpbz1BPpXFHz9m3CdqATR44Jvaydd" "[{"address": "t1M72Sfpbz1BPpXFHz9m3CdqATR44Jvaydd", "amount": 0.02 }]" 0 0.00002

Unfortunately this method doesn't work.
